public/css/list/item-table.less

```diff
@@ -62,3 +62,27 @@
     width: 1em;
   }
 }
+
+.controls .user-list,
+.controls .usergroup-list {
+  .usergroup-ball,
+  .user-ball {
+    height: 1.5em;
+    width: 1.5em;
+    line-height: ~"calc(1.5em - 4px)";
+    display: block;
+  }
+
+  .title br {
+    display: none;
+  }
+
+  .list-item > .col {
+    padding: .25em 0;
+  }
+}
+
+.hostgroup-list > col.object-statistics-graph,
+.servicegropup-list > col.object-statistics-graph {
+  padding-right: .25em;
+}
```

Comment:
Also doesn't belong here. Put that into `common.less` where all the other hostgroup- and servicegroup-list rules are.

Reply:
✅